src/libraries/System.Private.CoreLib/src/System/Threading/ThreadPoolWorkQueue.cs

```diff
@@ -381,6 +381,17 @@ public int Count
             }
         }
 
+#if CORECLR
+        // This config var can be used to enable an experimental mode that may reduce the effects of some priority inversion
+        // issues seen in cases involving a lot of sync-over-async. See EnqueueForPrioritizationTest() for more information. The
+        // mode is experimental and may change in the future.
+        internal static readonly bool s_prioritizationTest =
+            AppContextConfigHelper.GetBooleanConfig(
+                "System.Threading.ThreadPool.PrioritizationTest",
+                "DOTNET_ThreadPool_PrioritizationTest",
```

Comment:
Do we have other experimental flags? Do we call them "XxTest"? Just wondering about the naming.

Reply:
I didn't find other flags like this one. There are other unsupported ones but nothing special in the config var name itself. I just wanted to convey that this is a flag for testing things, its behavior may change, or it may be removed in the future.

Reply:
Ok. Maybe "Experiment" instead of "Test".